Src/AutoMoq/MockVirtualMethodsCommand.cs

```diff
@@ -115,6 +116,14 @@ private static IEnumerable<MethodInfo> GetConfigurableMethods(Type type)
                               ? type.GetInterfaceMethods()
                               : type.GetMethods();
 
+            // Skip properties that have both getters and setters to not interfere
+            // with other post-processors in the chain which initialize them."
+            var getSetMethods = type.GetProperties()
+                                    .Where(p => p.GetGetMethod() != null &&
+                                                p.GetSetMethod() != null)
+                                    .SelectMany(p => new[] {p.GetGetMethod(), p.GetSetMethod()});
```

Comment:
Thanks for that fix, it looks better now 👍 I think that these filters are internal to the class, so it's fine to rely that the subsequent filter will skip all the setters and extra safety is not required.

> Thought it would be more clear what I improved

It's often a dilemma what to keep in the history. Usually, I try to not pollute the history with intermediate decisions if I think they bring no value (e.g. I change name of the variable - nobody will be interested how it was initially named). In this case it could be considered the from different perspectives, that's why I _suggested_ to rebase - do what you feel is better 😉

BTW, the variable name `getSetMethods` is no longer valid as only `get` methods are present there..